classes/Task/AbstractTask.php

```diff
@@ -180,27 +180,7 @@ public function setErrorFlag(): void
         }
     }
 
-    /**
-     * @throws Exception
-     */
-    public function init(): void
-    {
-        $this->container->initPrestaShopCore();
-        $this->setupEnvironment();
-    }
-
-    /**
-     * @throws Exception
-     */
-    protected function setupEnvironment(): void
-    {
-        $updateConfiguration = $this->container->getUpdateConfiguration();
-
-        if ($this::TASK_TYPE === TaskType::TASK_TYPE_UPDATE && $updateConfiguration->isChannelLocal()) {
-            $archiveXml = $updateConfiguration->getLocalChannelXml();
-            $this->container->getFileLoader()->addXmlMd5File($this->container->getUpgrader()->getDestinationVersion(), $this->container->getProperty(UpgradeContainer::DOWNLOAD_PATH) . DIRECTORY_SEPARATOR . $archiveXml);
-        }
-    }
+    public function init(): void {}
```

Comment:
I think you can change this to 
```suggestion
   abstract public function init(): void;
```

Reply:
By declaring the method as abstract, I will need to define it on each task even if I have nothing to call. This is not the behavior I wanted.